prime-router/settings/prod/0114-hcintegrations-ca.yml

```diff
@@ -0,0 +1,305 @@
+# Run this:  ./prime multiple-settings set --input ./settings/prod/0114-hcintegrations-ca.yml --env prod
+---
+- name: "ca-dph"
+  description: "California Department of Public Health"
+  jurisdiction: "STATE"
+  stateCode: "CA"
+  countyName: null
+  filters: null
+  senders: []
+  receivers:
+  - name: "elr"
+    organizationName: "ca-dph"
+    topic: "covid-19"
+    customerStatus: "active"
+    translation: !<HL7>
+      schemaName: "covid-19"
+      useTestProcessingMode: false
+      useBatchHeaders: true
+      messageProfileId: null
+      replaceValue:
+        OBX-17-2: "OBX-17-9"
+        OBX-23-1: "OBX-23-1,-,OBX-23-10"
+      replaceValueAwithB: {}
+      replaceValueAwithB:
+        MSH-3: [ "*": "CDC PRIME - Atlanta,^2.16.840.1.114222.4.1.237821^ISO" ]
+        MSH-5: [ "*": "CDPH CA REDIE^2.16.840.1.114222.4.3.3.10.1.1^ISO" ]
+        MSH-6: [ "*": "CDPH_CID^2.16.840.1.114222.4.1.214104^ISO" ]
+      reportingFacilityName: null
+      reportingFacilityId: null
+      reportingFacilityIdType: null
+      suppressQstForAoe: false
+      suppressHl7Fields: "OBX-17-1, OBX-18-1, OBX-18-2, OBX-18-3, OBX-18-4"
+      suppressAoe: true
+      defaultAoeToUnknown: false
+      replaceUnicodeWithAscii: false
+      useBlankInsteadOfUnknown: null
+      truncateHDNamespaceIds: true
+      truncateHl7Fields: "OBX-23-1, ORC-21-1, OBR-16-3, ORC-12-3, PID-5-1, PID-5-2,\
+        \ PID-5-3"
```

Comment:
This is just question.   For line 38, Is this the way we continue line?

Reply:
Yes, I've seen these in more settings lately. I believe they're added automatically.